app/javascript/crayons/Modal/Modal.jsx

```diff
@@ -18,6 +18,7 @@ export const Modal = ({
   showHeader = true,
   sheetAlign = 'center',
   backdropDismissible = false,
+  allowOverflow = false,
```

Comment:
This customisation and the related CSS is needed to allow the datepicker calender to appear outside of the modal area (as it's wider)